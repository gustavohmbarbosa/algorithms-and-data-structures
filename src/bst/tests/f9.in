1
50
1
70
1
30
1
20
1
40
1
80
9
20
2
1
20
2
9
70
2
1
70
2
9
80
2
1
80
2
1
60
9
50
2
9
20
2
1
20
2
9
70
2
1
70
2
9
80
2
1
80
2
1
65
9
60
2
99