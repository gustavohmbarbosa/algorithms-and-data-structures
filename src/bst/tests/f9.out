[50][30][40][70][80]
[50][30][20][40][70][80]
[50][30][20][40][80]
[50][30][20][40][80][70]
[50][30][20][40][70]
[50][30][20][40][70][80]
[60][30][20][40][70][80]
[60][30][40][70][80]
[60][30][20][40][70][80]
[60][30][20][40][80]
[60][30][20][40][80][70]
[60][30][20][40][70]
[60][30][20][40][70][80]
[65][30][20][40][70][80]
